.github/workflows/prepare-release.yaml

```diff
@@ -20,7 +19,7 @@ jobs:
       - name: Setup go
         uses: actions/setup-go@v3
         with:
-          go-version: '1.19'
+          go-version: "1.19"
```

Comment:
is that necessary ?

Reply:
Oh, my YAML linter/formatter must've caught that. Apparently it thinks that you should use double quotes. I can revert it if you prefer?